.vscode/settings.json

```diff
@@ -25,5 +25,9 @@
   "files.eol": "\n",
   "typescript.enablePromptUseWorkspaceTsdk": true,
   "typescript.tsdk": "node_modules/typescript/lib",
-  "eslint.workingDirectories": [{ "pattern": "packages/*/" }]
+  "eslint.workingDirectories": [
+    {
+      "pattern": "packages/*/"
+    }
+  ]
```

Comment:
strange this formatting would have changed cc/ @auscompgeek

Reply:
:shrug: looks like prettier is fine with either too? weird that this is the _only_ change to this file

Reply:
I removed this change.